EDU-1521 Release stages guide (#2649)
temporalio/documentation: docs-src/product/release-stages/public-preview.md

---
id: public-preview
title: Public Preview
description: Learn more about the Public Preview release stage
sidebar_label: Public Preview
tags:
  - product release stages
  - term
---

**Access:** Public preview of New Features are available to everyone.

**Classification:** Public preview of New Features may be further developed and tested.

> These New Features are undergoing further refinement and are recommended for production usage.

Reviewer: Are you missing a "not" regarding production usage? (It may just be my misunderstanding.)

Reviewer 2: No, they are recommended for prod

> These New Features are undergoing further refinement and are recommended for production usage.

Reviewer 2: ```suggestion
These New Features are being refined and are recommended for production usage.
```

repetition

> There may be exceptions for different features, this is the typical expectation.

Reviewer 2: ```suggestion
There may be exceptions for different features, but this is the typical expectation.
```

> Any variation will be covered in documentation.

Reviewer: Mild rephrasing suggestion: "Any variation will be documented."

Reviewer 2: ```suggestion
Any variation will documented.
```

> **Feedback:** Temporal encourages Temporal Cloud users to submit feedback directly through the [Temporal Support Team](/cloud/support-create-ticket).
> This feedback will assist in guiding the improvements for General Availability.
> Temporal self-hosted users are invited to share feedback via the Community Slack channel; by reaching out directly to the Temporal Product team at product@temporal.io, or by logging issues on the appropriate [GitHub repository](https://github.com/temporalio).

Reviewer 2: ```suggestion
**Feedback:** Temporal users are invited to share feedback via the [Community Slack](http://t.mp/slack), by reaching out directly to the Temporal Product team at product@temporal.io, or by creating issues in the relevant [GitHub repository](https://github.com/temporalio).
Temporal encourages Temporal Cloud users to submit feedback [via support ticket](/cloud/support-create-ticket).
This feedback will assist in guiding the improvements for General Availability.
```

- oss first
- via support team feels less specific than "open a ticket", eg they're on the #support channel.

> **Availability:** New Features in public preview may evolve. The APIs may undergo changes however, Temporal’s goal is to maintain backward compatibility.

Reviewer 2: ```suggestion
**Availability:** New Features in Public Preview may evolve. The APIs may undergo changes; however, Temporal’s goal is to maintain backward compatibility.
```
